middlewares/auth.js

```diff
@@ -5,7 +5,6 @@ const { JWT_SECRET } = require('../configuration/settings');
 module.exports = (req, res, next) => {
   const token = req.cookies.jwt;
   let playload;
```

Comment:
**можно улучшить:**  похоже, что тут опечатка

Reply:
где опечатка?не совсем понимаю о чем речь? модуль работает как надо)

Reply:
Просто опечатка в названии переменной, в дальнейшем она используется в таком же виде и никак не влияет на работоспособность кода

```suggestion
  let payload;
```